runtimes/standalone/src/lib.rs

```diff
@@ -269,6 +269,7 @@ parameter_types! {
 }
 
 impl pallet_transaction_payment::Config for Runtime {
+	type Event = Event;
```

Comment:
Interesting to note: For standalone we do a lazy assignment of the `TransactionPayment` module:
```
TransactionPayment: pallet_transaction_payment = 7,
```

For the parachain runtimes we don't. That's why we did not have to add the `Event<T>` type as was done for Spiritnet and Peregrine.